istio-operator/istio-operator.spec

```diff
@@ -103,13 +107,20 @@ popd
 mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
 pushd OPERATOR/src/github.com/maistra/istio-operator/tmp/_output/
 cp -rpav helm/ $RPM_BUILD_ROOT%{_sysconfdir}/istio-operator/%{charts_version}
+popd
 
+#install manifests
+install -d $RPM_BUILD_ROOT%{_datadir}/%{name}
+cp -ra OPERATOR/src/github.com/maistra/istio-operator/manifests $RPM_BUILD_ROOT%{_datadir}/%{name}
 
 %files
 %{_bindir}/istio-operator
 %{_sysconfdir}/istio-operator
+%{_datadir}/%{name}/manifests
```

Comment:
I though you said these files should reside in /manifests

Reply:
Nods, in the image. I suppose I could install them in the "/", it just feels super-weird?

Reply:
Updated to install manifest files into "/manifests"